dotCMS/src/main/java/com/dotcms/publisher/bundle/business/BundleAPIImpl.java

```diff
@@ -391,4 +398,39 @@ public void deleteAssetFromBundle(String assetId, String bundleId)
 
 	}
 
+	
+
+    @CloseDBIfOpened
+    @Override
+    public File generateBundleDirectory(Bundle bundle) {
+
+        final PushPublisherConfig pushPublisherConfig = new PushPublisherConfig(bundle);
+        pushPublisherConfig.setPublishers(Arrays.asList(GenerateBundlePublisher.class));
+        try {
+            APILocator.getPublisherAPI().publish(pushPublisherConfig);
+        }
+        catch(Exception e) {
```

Comment:
Codacy found an issue: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6239861)